python-package/lightgbm/basic.py

```diff
@@ -1721,15 +1721,23 @@ def save_model(self, filename, num_iteration=None):
             Index of the iteration that should be saved.
             If None, if the best iteration exists, it is saved; otherwise, all iterations are saved.
             If <= 0, all iterations are saved.
+        start_iteration: int, optional (default=0)
+            Start index of the iteration that should to saved.
         """
         if num_iteration is None:
             num_iteration = self.best_iteration
         _safe_call(_LIB.LGBM_BoosterSaveModel(
             self.handle,
+            ctypes.c_int(start_iteration),
             ctypes.c_int(num_iteration),
             c_str(filename)))
         _save_pandas_categorical(filename, self.pandas_categorical)
 
+    def shuffle_models(self):
+        """Shuffle models
+        """
+        _safe_call(_LIB.LGBM_BoosterShuffleModels(self.handle))
+
     def _load_model_from_string(self, model_str, verbose=True):
```

Comment:
Don't you want to make `_load_model_from_string` publicly available, since `model_to_string` has become public?